Changelogs for Digital Twin Manager Microservice 1024.1.0 (#4235)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1024-1-0-fixed-updating-the-title-and-description-of-library-asset-definitions.md

---
date: ""

> title: "Fixed updating the title and description of library Asset Definitions"

Reviewer: ```suggestion
title: "Fixed updating the title and description of library asset definitions"
```

> title: "Fixed updating the title and description of library asset definitions"

Reviewer: ```suggestion
title: "Custom titles and descriptions of library asset definitions no longer replaced with default values"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2642"
version: "1024.1.0"
---

> Previously, there was an issue with the update of the library Asset Definitions which replaced a custom title and description of such a Definition with the default values after the microservice was resubscribed. This is fixed now.

Reviewer: ```suggestion
Previously, custom titles and descriptions of library asset definitions were replaced with the default values after the microservice was resubscribed. This issue has been fixed.
```

---
date: ""
title: "Fixed updating the title and description of library Asset Definitions"
product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2642"
version: "1024.1.0"
---
Previously, there was an issue with the update of the library Asset Definitions which replaced a custom title and description of such a Definition with the default values after the microservice was resubscribed. This is fixed now.

Co-authored-by: Frédéric Joulin <53534562+frej-c8y@users.noreply.github.com>
Co-authored-by: Michael Voigt <91879843+mvoigt-sag@users.noreply.github.com>
Co-authored-by: Nam Nhat Pham <namnhat.pham@cumulocity.com>

> Co-authored-by: Michael Voigt <michael.voigt@cumulocity.com>

Reviewer 2: ```suggestion
```
